<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tDialog</class>
 <widget class="QDialog" name="tDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>GOST type A</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="teacherLastnameLabel">
       <property name="text">
        <string>Фамилия преподавателя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="teacherLastnameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="teacherNameLabel">
       <property name="text">
        <string>Имя преподавателя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="teacherNameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="teacherSurnameLabel">
       <property name="text">
        <string>Отчество преподавателя:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="teacherSurnameEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="10,40">
     <item>
      <widget class="QLabel" name="ratingLabel">
       <property name="text">
        <string>Рейтинг:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="ratingSetter">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="10,20">
     <item>
      <widget class="QLabel" name="amountOfSubjectsLabel">
       <property name="text">
        <string>Число предметов:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="amountOfSubjectsSetter">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>5</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="confirm">
       <property name="text">
        <string>Ок</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
